### DISCUSSION TITLE
Is this project still in development?

### DISCUSSION THREAD

**Discussion by Connor56** [rhasspy/piper#656] [General]
The commit history doesn't show many changes. Is that because the project is "feature complete" so to speak? Or has development stopped for some reason?

**Comment by synesthesiam** [comment]
I've made good progress on a rewrite of Piper that (1) does not use espeak-ng due to licensing concerns and (2) has a C API for better interoperability, and (3) supports SSML.

Development has stalled, unfortunately, due to a lack of time. I've been hesitant to make changes to this repo (even "quick" bug fixes) for fear of making things worse without taking the proper time to get back into the right mindset.
 
I would like to at least fix the outstanding wheel issues so it's easier to install. I may try bumping dependencies as well to fix the 4090 problem, but that often leads to many more issues because of Python's dependency management mess.

**Reply by Connor56** [reply]
Okay, cool. Thank you for the update!

**Comment by wukonggeo** [comment]
@synesthesiam
Thank you for your efforts. Perhaps you could release some smaller test versions first, allowing the community to get involved and help shape the final release?

**Comment by TheStigh** [comment]
(following)
